client/rest-client/src/main/java/org/infinispan/client/rest/RestServerClient.java

```diff
@@ -36,4 +36,6 @@
    CompletionStage<RestResponse> unIgnoreCache(String cacheManagerName, String cacheName);
 
    CompletionStage<RestResponse> listIgnoredCaches(String cacheManagerName);
+
+   RestLoggingClient logging();
```

Comment:
why in `RestServerClient`? Shouldn't the logging affect all nodes in the cluster? 
maybe under `RestClient` or `RestClusterClient`?